homeassistant/components/advantage_air/entity.py

```diff
@@ -30,6 +31,7 @@ def update_handle_factory(self, func, *keys):
         async def update_handle(*values):
             try:
                 if await func(*keys, *values):
+                    await asyncio.sleep(0.2)
```

Comment:
Coordinator has a way for this instead of just adding a sleep. 
Alternatively you can write the value to the data instead and skip this additional refresh (as it seems you know it's successful or not).

Reply:
On the coordinator you can set a debouncer.
You can look at the coordinator in the `sensibo` integration as example.
Also use `async_request_refresh()` and not `async_refresh()` as it executes immediately were the first one waits a bit to batch if possible.